From 94a8d706415e0ab319b5b555761711977d3ab5c9 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Fri, 14 Jul 2017 11:34:36 -0700
Subject: [PATCH 28/44] Make timesyncd a "simple" service

the daemon ends up waiting 100 msec on networking conditions before notifying
that it is done, even though that's really not relevant
---
 units/systemd-timesyncd.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-timesyncd.service.in b/units/systemd-timesyncd.service.in
index 92ee94582c..a704bcebc5 100644
--- a/units/systemd-timesyncd.service.in
+++ b/units/systemd-timesyncd.service.in
@@ -45,7 +45,7 @@ StateDirectory=systemd/timesync
 SystemCallArchitectures=native
 SystemCallErrorNumber=EPERM
 SystemCallFilter=@system-service @clock
-Type=notify
+Type=simple
 User=systemd-timesync
 @SERVICE_WATCHDOG@
 
-- 
2.28.0

